integration/docker/tag_test.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2018 Intel Corporation
```

Comment:
@GabyCT I went through the `docker tag` documentation and it does not involve running containers. I think it doesn't make sense to have the test here here since we want the integration tests to focus on the runtime. What do yoiuthink?